src/panel/details.tsx

```diff
@@ -63,8 +64,10 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
 				<div className="ellipsis">{text ?? '—'}</div>
 				<div className="svSecondary">{uri?.file ?? '—'}</div>
 				<div className="svLineNum">{region?.startLine}:1</div>
-			</>;
-		};
```

Comment:
Outside of this PR scope, but what's going on with the whitespace here?

Reply:
Weird. pushed a fix in this PR